development.md

```diff
@@ -0,0 +1,24 @@
+### Get  Started with Development 
+#
+
+## prerequisites
```

Comment:
This is a good start, instead of providing details on all of the prerequisites, we can shorten this section into just a bullet list with links to those products pages. example.

```
## Prerequisites
- [go](https://go.dev/dl/)
- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
ect.
```

Additionally, let's not list docker as a prerequisite here since it's not directly our dependency but a dependency of kind. I think it's best to only list the prerequisites that we have and not try and capture the entire dependency tree if that makes sense. 

Lastly, try and keep markdown headings capitalized so change `prerequisites` to `prerequisites` .

Reply:
done

Reply:
i have raised a new pr for makefile modification: https://github.com/kubernetes-sigs/hydrophone/pull/110